Robot Code/src/org/usfirst/frc/team3316/robot/subsystems/Chassis.java

```diff
@@ -210,29 +216,55 @@ public boolean isOnDefense()
 	// navX Class
 	private class navX extends TimerTask
```

Comment:
Move this class to the start of the document. Usually it is more coherent when nested classes come before the content of the outer class.